static/search.js

```diff
@@ -305,15 +308,27 @@ var Search = Control.extend({
 	//   else
 	//     generates search engine from searchMap & saves index to local storage
 	initSearchEngine: function(searchMap){
-		var localStorageKey = this.formatLocalStorageKey(this.searchIndexLocalStorageKey),
-				index = this.getLocalStorageItem(localStorageKey);
-		if(index){
-			this.searchEngine = searchEngine.Index.load(index);
+		var searchEngine;
+		var searchIndexKey = this.formatLocalStorageKey(this.searchIndexLocalStorageKey);
+		var searchIndexVersionKey = this.formatLocalStorageKey(this.searchIndexVersionLocalStorageKey);
+		var index = this.getLocalStorageItem(searchIndexKey);
+		var indexVersion = this.getLocalStorageItem(searchIndexVersionKey);
+		var currentIndexVersion = 1;// Bump this whenever the index code is changed
```

Comment:
See this commit for more details: https://github.com/canjs/bit-docs-html-canjs/commit/0d346e5b32244c61d9063c104cad53ac85d9b96f